executor/testdata/executor_suite_out.json

```diff
@@ -598,5 +598,1400 @@
+        "Plan": [
+          "TableReader 250.00 root partition:p0,p1 data:Selection",
+          "└─Selection 250.00 cop[tikv]  ge(testrangepartitionboundariesbetweenm.t.a, 999999), le(testrangepartitionboundariesbetweenm.t.a, 1000009)",
+          "  └─TableFullScan 10000.00 cop[tikv] table:t keep order:false, stats:pseudo"
+        ],
+        "Res": [
+          "1000000 1000000 Filler ...",
+          "1000001 1000001 Filler ...",
+          "1000002 1000002 Filler ...",
+          "999999 999999 Filler ..."
+        ]
+      },
+      {
+        "SQL": "SELECT * FROM t WHERE a BETWEEN 999999 AND 1999997",
+        "Plan": [
+          "TableReader 250.00 root partition:p0,p1 data:Selection",
+          "└─Selection 250.00 cop[tikv]  ge(testrangepartitionboundariesbetweenm.t.a, 999999), le(testrangepartitionboundariesbetweenm.t.a, 1999997)",
+          "  └─TableFullScan 10000.00 cop[tikv] table:t keep order:false, stats:pseudo"
+        ],
+        "Res": [
+          "1000000 1000000 Filler ...",
+          "1000001 1000001 Filler ...",
+          "1000002 1000002 Filler ...",
+          "999999 999999 Filler ..."
+        ]
+      },
+      {
+        "SQL": "SELECT * FROM t WHERE a BETWEEN 999999 AND 1999998",
+        "Plan": [
+          "TableReader 250.00 root partition:p0,p1 data:Selection",
+          "└─Selection 250.00 cop[tikv]  ge(testrangepartitionboundariesbetweenm.t.a, 999999), le(testrangepartitionboundariesbetweenm.t.a, 1999998)",
+          "  └─TableFullScan 10000.00 cop[tikv] table:t keep order:false, stats:pseudo"
+        ],
+        "Res": [
+          "1000000 1000000 Filler ...",
+          "1000001 1000001 Filler ...",
+          "1000002 1000002 Filler ...",
+          "1999998 1999998 Filler ...",
+          "999999 999999 Filler ..."
+        ]
+      },
+      {
+        "SQL": "SELECT * FROM t WHERE a BETWEEN 999999 AND 1999999",
+        "Plan": [
+          "TableReader 250.00 root partition:p0,p1 data:Selection",
+          "└─Selection 250.00 cop[tikv]  ge(testrangepartitionboundariesbetweenm.t.a, 999999), le(testrangepartitionboundariesbetweenm.t.a, 1999999)",
+          "  └─TableFullScan 10000.00 cop[tikv] table:t keep order:false, stats:pseudo"
+        ],
+        "Res": [
+          "1000000 1000000 Filler ...",
+          "1000001 1000001 Filler ...",
+          "1000002 1000002 Filler ...",
+          "1999998 1999998 Filler ...",
+          "1999999 1999999 Filler ...",
+          "999999 999999 Filler ..."
+        ]
+      },
+      {
+        "SQL": "SELECT * FROM t WHERE a BETWEEN 999999 AND 2000000",
+        "Plan": [
+          "TableReader 250.00 root partition:all data:Selection",
+          "└─Selection 250.00 cop[tikv]  ge(testrangepartitionboundariesbetweenm.t.a, 999999), le(testrangepartitionboundariesbetweenm.t.a, 2000000)",
+          "  └─TableFullScan 10000.00 cop[tikv] table:t keep order:false, stats:pseudo"
+        ],
+        "Res": [
+          "1000000 1000000 Filler ...",
```

Comment:
The result is not ordered, so the test may be unstable?

Reply:
It should be ordered, according to the sort function here `tk.MustQuery(tt).Sort().Check(testkit.Rows(output[i].Res...))`